starfish/image/_filter/bandpass.py

```diff
@@ -25,7 +25,7 @@ def __init__(
         threshold : float
             zero any pixels below this intensity value
         truncate : float
-            # TODO dganguli: this is not documented by trackpy, can you help?
+            truncate the filter at this many standard deviations
```

Comment:
truncate the gaussian kernel, used by the gaussian filter, by this many standard deviations.